### DISCUSSION TITLE
Parser generator documetation

### DISCUSSION THREAD

**Discussion by svex99** [alecthomas/participle#273] [General]
Hi, thanks for this amazing library!

I'm currently using participle in a project and I need to document in the most formal way the parser generated. Will be great to know the type of parser being generated for the library and, if possible, the method used in the generation. Any further read or reference will be very useful.

Thanks in advance!

**Comment by alecthomas** [comment]
It is described in the README [here](https://github.com/alecthomas/participle#limitations). There is no parser code *generator* in Participle currently either, FWIW.
